Carousel/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
+  <script src="https://kit.fontawesome.com/4ca0456cfc.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="container">
+
+    <img class="picture" src="./img/img0.jpg" alt="image">
+    <img class="picture" src="./img/img1.jpg" alt="image">
+    <img class="picture" src="./img/img2.jpg" alt="image">
+    <img class="picture" src="./img/img3.jpg" alt="image">
+    <img class="picture" src="./img/img4.jpg" alt="image">
+    <img class="picture" src="./img/img5.jpg" alt="image">
+
+    <i id="prev" class="fa-solid fa-chevron-left" onclick="showImage(-1)"></i>
+    <i id="next" class="fa-solid fa-chevron-right" onclick="showImage(+1)" ></i>
+
+    <div class="dots">
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+    </div>
+
```

Comment:
as i described before

we don't add an empty line before closing an element

Reply:
fixed